### DISCUSSION TITLE
Unturned Server Host

### DISCUSSION THREAD

**Discussion by FumetsuThe1** [SmartlyDressedGames/Unturned-3.x-Community#2489] [General]
Getting this problem as of 20/03/2021. I'm hosting a server but I cannot seem to join it. Tried hosting LAN and Internet but neither work. It says "Lost connection to host or Steam network." I am port forwarding ports 27015 - 27017. Validated my games files, added the acceptions to my firewall, my antivirus should be disabled, yet I still cannot join. I'm joining from the LAN tab and I'm using the SteamCMD method. Restarted the server and tried multiple and it will not work. And I have my addons disabled. Here's what pops up when I try to join. 
![image](https://user-images.githubusercontent.com/81042075/111882295-2251df00-8972-11eb-9a4b-a666d2dfa21c.png)

Thanks for any help given! Even if it doesn't work! (also I'd reccomend a bug discussion category)

**Comment by SDGNelson** [comment]
Recently this has been popping up occasionally. Yesterday it happened on my test server and for some reason steam_appid.txt was 1110390. If you open U3DS/steam_appid.txt does it say 304930? (it should) This file is included in the Steam download, but I think I might change the game to write it before initializing Steam.

**Reply by FumetsuThe1** [reply]
Thanks! Now I can join my own server but my friend still cannot.

**Reply by SDGNelson** [reply]
What happened when your friend tried to join? Did you port forward? https://github.com/SmartlyDressedGames/U3-Docs/blob/master/PortForwarding.md

**Reply by FumetsuThe1** [reply]
Yeah. I forwarded ports 27015, 27016, and 27017. Both on TCP and UDP. And oppened the ports in firewall. And my antivirus is uninstalled too. Whenever they join it says "Failed to find server" and nothing about them trying to join pops up in server console. I was guessing it was a port forwarding issue but had no idea how to fix it.

**Reply by SDGNelson** [reply]
"Failed to find server" means that the Steam server details query failed. Most likely meaning the port forwarding is incorrect. Did you set a different port on the command-line or in Commands.dat? By default it will be 27015.

**Reply by FumetsuThe1** [reply]
I have it set in Commands.dat.

**Reply by SDGNelson** [reply]
What did you set it to? With those ports forwarded it should be set to 27015.

**Reply by FumetsuThe1** [reply]
Yeah it's set to 27015

**Reply by SDGNelson** [reply]
In that case it must be related to the port forwarding setup. Sorry that I cannot suggest anything more useful. Since you forwarded the correct ports my guess would be that the local IP entered into your router is incorrect.

**Reply by FumetsuThe1** [reply]
It's fineeee. Thanks anways though!
